tfrecorder/schema.py

```diff
@@ -88,52 +96,48 @@ def get_raw_schema_map(
   """Converts a schema to a raw (pre TFT / post image extraction) schema."""
   raw_schema = {}
   for k, v in schema_map.items():
-    if v.type_name == "image_uri":
-      raw_schema['image_name'] = image_support_type
-      raw_schema['image'] = image_support_type
-      raw_schema['image_height'] = image_support_type
-      raw_schema['image_width'] = image_support_type
-      raw_schema['image_channels'] = image_support_type
+    if v.type_name == 'image_uri':
+      raw_schema['image_name'] = image_support_string
```

Comment:
What differentiates `image_support_string/int` from another int type? How about using `integer_input` and `string_input`?

Reply:
I made them separate types because I didn't want any preprocessing to be done on them.  For example, if we made the choice in the future to integerize strings, we wouldn't want this string to be integerized. I meant these to be internal types that preprocessing_fn wouldn't modify.